packages/core/src/util/nodeJsVersionCheck.js

```diff
@@ -17,8 +17,11 @@ exports.minimumNodeJsVersion = 14;
  */
 exports.isNodeJsTooOld = function isNodeJsTooOld() {
   const currentVersion = process.version;
+
   if (typeof currentVersion === 'string') {
-    const majorVersionStr = process.version.split('.')[0];
+    const baseVersion = currentVersion?.split('-')[0];
```

Comment:
explain why?

Reply:
It was failing for some test cases. but will be removing to test case resolution pr if needed